examples/shared-examples/usage-reporting/cluster-connector.yaml

```diff
@@ -14,65 +14,76 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: nginx-cluster-connector
 rules:
-  - apiGroups:
-      - ""
-    resources:
-      - pods
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - secrets
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - nodes
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - namespaces
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-    resources:
-      - events
-    verbs:
-      - create
-      - patch
-      - list
-  - apiGroups:
-      - coordination.k8s.io
-    resources:
-      - leases
-    verbs:
-      - get
-      - list
-      - watch
-      - update
-      - create
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - list
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - create
+- apiGroups:
+  - security.openshift.io
+  verbs:
+  - create
+  - delete
+  - get
+  - use
+  resources:
+  - securitycontextconstraints
+  resourceNames:
+  - nginx-cluster-connector
```

Comment:
Aside from adding this new `apiGroup` it looks like the indentation of this section was adjusted. Does it still deploy okay?

Reply:
it does

Reply:
Yep, just brought the indentation in-line with NIC rbac files.


```
~/nginx/kubernetes-ingress on chore/update-cluster-connector-docs-ex ● ● λ kaf examples/shared-examples/usage-reporting/cluster-connector.yaml
namespace/nginx-cluster-connector unchanged
serviceaccount/nginx-cluster-connector created
clusterrole.rbac.authorization.k8s.io/nginx-cluster-connector created
clusterrolebinding.rbac.authorization.k8s.io/nginx-cluster-connector created
deployment.apps/nginx-cluster-connector created
ve.patel in ~/nginx/kubernetes-ingress on chore/update-cluster-connector-docs-ex ● ● λ k get po -n nginx-cluster-connector 
NAME                                       READY   STATUS    RESTARTS   AGE
nginx-cluster-connector-6b98b69ccb-hkd67   1/1     Running   0          6s
ve.patel in ~/nginx/kubernetes-ingress on chore/update-cluster-connector-docs-ex ● ● λ klogf -n nginx-cluster-connector nginx-cluster-connector-6b98b69ccb-hkd67 
I0703 09:45:43.385431       1 main.go:95] Starting with flags: ["-nms-server-address=https://apigw.nms.svc.cluster.local:443/api/platform/v1" "-nms-basic-auth-secret=nginx-cluster-connector/nms-basic-auth"]
I0703 09:45:43.385790       1 main.go:102] Starting NGINX Cluster Connector Version: 0.2.0 Commit: 7fc0259bf716a683f033b29b59d22650149e7a5c Date: 2024-07-02T15:26:44Z DirtyState: false Arch: linux/amd64 Go: go1.22.4
I0703 09:45:43.386814       1 main.go:45] Starting with NMS address: https://apigw.nms.svc.cluster.local:443/api/platform/v1
E0703 09:45:43.399938       1 k8s.go:153] error retrieving secret: secrets "nms-basic-auth" not found
E0703 09:45:43.400471       1 connect.go:50] Failed to create basic auth string: invalid secret type. Expected kubernetes.io/basic-auth
I0703 09:45:43.416813       1 k8s.go:116] Number of nodes in cluster: 3
E0703 09:45:43.423080       1 connect.go:216] Request failed: empty credentials
I0703 09:45:43.423160       1 connect.go:132] Failed to get cluster info: empty credentials
I0703 09:45:43.425742       1 k8s.go:79] Kubernetes version: 1.29.6
I0703 09:45:43.426819       1 main.go:124] Starting cluster connector control loop
I0703 09:45:43.426866       1 controller.go:143] Starting cluster agent control loop
E0703 09:45:43.627872       1 controller.go:242] no basic auth basicAuthSecret found in current cluster state
```